pkg/kubelet/dockertools/manager.go

```diff
@@ -47,6 +47,7 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/prober"
 	"k8s.io/kubernetes/pkg/kubelet/qos"
 	kubeletTypes "k8s.io/kubernetes/pkg/kubelet/types"
+	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
```

Comment:
I can't find anything that says this is a problem, and I don't think we
should be doing manual edits on a branch unless we HAVE TO.

On Mon, Oct 26, 2015 at 11:06 PM, Dawn Chen <notifications@github.com>
wrote:

> In pkg/kubelet/dockertools/manager.go
> <https://github.com/kubernetes/kubernetes/pull/16314#discussion_r43086871>
> :
>
> > @@ -47,6 +47,7 @@ import (
> >  	"k8s.io/kubernetes/pkg/kubelet/prober"
> >  	"k8s.io/kubernetes/pkg/kubelet/qos"
> >  	kubeletTypes "k8s.io/kubernetes/pkg/kubelet/types"
> > +	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
>
> Master doesn't have a dup import. The initial alias used here is
> kubeletTypes, which doesn't match the naming convention for go package. We
> fixed it later. I believe the 1.1 was cut between, and such cleanup code
> wasn't cherry-picked.
>
> Should we fix this in 1.1 branch by cherry-pick the cleanup code too?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/16314/files#r43086871>.
>